src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_pix2pix_zero.py

```diff
@@ -562,6 +612,41 @@ def get_embeds(self, prompt: List[str], batch_size: int = 16) -> torch.FloatTens
 
         return torch.cat(embeds, dim=0).mean(0)[None]
 
+    def prepare_image_latents(self, image, batch_size, dtype, device, generator=None):
```

Comment:
If it's copied from an existing pipeline let's add the `# Copied from ...` statement. Also, does the image need to be scaled before it passed to this function? If so, is that noted somewhere?

Reply:
yeah this one is a bit different actually